boa_engine/src/context/icu.rs

```diff
@@ -4,8 +4,10 @@ use icu_collator::{Collator, CollatorError, CollatorOptions};
 use icu_list::{ListError, ListFormatter, ListLength};
 use icu_locid_transform::{LocaleCanonicalizer, LocaleExpander, LocaleTransformError};
 use icu_provider::{
-    yoke::{trait_hack::YokeTraitHack, Yokeable},
-    zerofrom::ZeroFrom,
+    prelude::{
+        yoke::{trait_hack::YokeTraitHack, Yokeable},
+        zerofrom::ZeroFrom,
+    },
```

Comment:
This change is required due to backwards in-compatibility, that might be fixed in https://github.com/unicode-org/icu4x/pull/3333, but we might just want to move to explicitly import `yoke` and `zerofrom`, so that we don't rely on `#[doc(hidden)]` features.

Reply:
Yeah, it'll be better to import both deps directly.